docs/announcement.md

```diff
@@ -0,0 +1,120 @@
+We annouce both community-ee-base and community-ee-minimal together
+
+# Forum Announcement
+
+Release announcement to be done  in the  [Ecosystem Releases category](https://forum.ansible.com/c/news/releases)  under the News and Announcement section.
+
+# E-Mail Announcement
+
+`ansible-annouce` Google Groups
+
+# community-ee-base announcement
```

Comment:
Isn't it an announcement for *both* images?

Reply:
Yes this is an announcement for both. Fixing it.